ansible/roles/stack-sunbird/templates/sunbird_groups-service.env

```diff
@@ -18,6 +18,10 @@ sunbird_sso_url={{keycloak_auth_server_url}}/
 sunbird_sso_realm={{keycloak_realm}}
 sunbird_sso_client_id={{sunbird_sso_client_id}}
 sunbird_sso_client_secret={{sunbird_sso_client_secret | d('')}}
+sunbird_sso_username={{sunbird_sso_username}}
```

Comment:
@Hari-stackroute where is the default value for this ?

Reply:
Defaults are in all.yml